test/fixtures/include-path/lib-alternate/vars.scss

```diff
@@ -0,0 +1 @@
+$color: orange;
```

Comment:
I've tried to make my fixtures simpler and clearer. I hope it's the sort of thing you had in mind.